src/app/components/GearCardEdit/GearCardEdit.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import DeleteIcon from '../assets/DeleteIcon';
+import EditIcon from '../assets/EditIcon';
+import style from './GearCardEdit.module.css';
+
+export default function GearCardEdit(): JSX.Element {
```

Comment:
Maybe come up with a different name, aren't these the two button at the bottom part of your cards? I'd maybe call it `GearCardButtonSection`

Reply:
Resolved in [bfcc656](https://github.com/Seborider/plugs./pull/32/commits/bfcc6564958c9f5172a4eaa9243b945e1710465c)